app/build.gradle

```diff
@@ -121,6 +121,8 @@ dependencies {
     // Dagger
     implementation "com.google.dagger:dagger:$dagger_version"
     implementation "com.google.dagger:dagger-android-support:$dagger_version"
+    implementation 'androidx.appcompat:appcompat:1.0.2'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
```

Comment:
I forgot to comment that this should be separated from dagger's implementations, should be have a comment like `ConstraintLayout` before and have the version variable like the others.